tools/SourceKit/tools/sourcekitd/lib/API/Requests.cpp

```diff
@@ -313,10 +325,9 @@ void sourcekitd::handleRequest(sourcekitd_object_t Req,
   });
 }
 
-static std::unique_ptr<llvm::MemoryBuffer>
-getInputBufForRequest(Optional<StringRef> SourceFile,
-                      Optional<StringRef> SourceText,
-                      llvm::SmallString<64> &ErrBuf) {
+static std::unique_ptr<llvm::MemoryBuffer> getInputBufForRequest(
+    Optional<StringRef> SourceFile, Optional<StringRef> SourceText,
+    llvm::vfs::FileSystem *FileSystem, llvm::SmallString<64> &ErrBuf) {
```

Comment:
Looks like this is the only non-refcounted pointer to `llvm::vfs::FileSystem`? Would it make sense to change it to a refcounted pointer for consistency?